### DISCUSSION TITLE
Welcome to moeGramX Discussions!

### DISCUSSION THREAD

**Discussion by jplie** [moeCrafters/moeGramX#12] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by stepashka61** [comment]


**Reply by jplie** [reply]
С недавнего времени Телеграм, к сожалению, не отправляет код для входа через смс на неофициальные клиенты, поэтому единственный вариант это включить получение кода по почте или же получить код через официальное приложение.

**Reply by stepashka61** [reply]
> С недавнего времени Телеграм, к сожалению, не отправляет код для входа через смс на неофициальные клиенты, поэтому единственный вариант это включить получение кода по почте или же получить код через официальное приложение.

Прощу прощения, разобрался, классный софт делаете.
